cypress-tests/cypress/support/commands.js

```diff
@@ -443,17 +445,22 @@ Cypress.Commands.add("refundCallTest", (requestBody, refund_amount, det, globalS
       "Content-Type": "application/json",
       "api-key": globalState.get("apiKey"),
     },
+    failOnStatusCode: false,
     body: requestBody
   }).then((response) => {
     logRequestId(response.headers['x-request-id']);
 
     expect(response.headers["content-type"]).to.include("application/json");
     console.log(response.body);
     globalState.set("refundId", response.body.refund_id);
-    expect(response.body.status).to.equal(det.refundStatus);
-    expect(response.body.amount).to.equal(refund_amount);
-    expect(response.body.payment_id).to.equal(payment_id);
-  });
+    if (response.body.status === det.refundStatus) {
+      expect(response.body.status).to.equal(det.refundStatus);
+      expect(response.body.amount).to.equal(refund_amount);
+      expect(response.body.payment_id).to.equal(payment_id);
+    } else if (response.body.error.type === "invalid_request") {
+      expect(response.body.error).to.deep.equal(Errors.refundErrors["paymentStatusProcessing"]);
```

Comment:
This kind of assertion checks will be addressed in a separate PR.  
Thought is to do something like:

2xx -> success and connector error response handling
4xx -> error response handling
5xx -> error response handling